node/src/BaseClient.ts

```diff
@@ -113,16 +114,18 @@ import {
     createZRemRangeByRank,
     createZRemRangeByScore,
     createZScore,
-    createSUnion,
```

Comment:
How is it related?

Reply:
its a linter that fixes the imports to be in alphabetic order, I think you can see it in all prs of node and the first to be merged will fix it :)

Reply:
ack